Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDerivedCredential/settings.json

```diff
@@ -0,0 +1,25 @@
+{
+    "resourceName": "IntuneDerivedCredential",
+    "description": "Use this resource to create new navigation property to derivedCredentials for device Management in Intune.",
+
+    "permissions": {
+      "graph": {
+        "delegated": {
+          "read": [
+            ""
```

Comment:
Either leave this empty [] or provide a value but empty quotes is making the tests fail.

Reply:
updated and pushed.